src/components/Arcsi/Arcsi.tsx

```diff
@@ -19,7 +19,11 @@ export const Arcsi = (props: NativeStackScreenProps<any>) => {
     if (!shows) {
       return [];
     }
-    return shows.sort((a, b) => a.name.localeCompare(b.name, 'hu'));
+    return shows
+      .filter(item => item.items?.length > 0)
```

Comment:
@molnar-a I think you misunderstood the task, the Show itself should be in the list at Shows page, even if it doesn't have any archived episodes (items)
The good example for this is
https://lahmacun.hu/shows/lahmacun-gyorsnaszad-hidja or https://lahmacun.hu/shows/european-lab-budapest-x-sphera
What about this in `showDetail.tsx` ?
```
     ListHeaderComponent={
        <View>
          <Image style={styles.coverImage} source={{ uri: cover_image_url }} />
          <Text style={styles.showName}>{name}</Text>
          <Text style={styles.showDescription}>{description}</Text>
          {orderedItems.length ? (
            <>
              <Text style={styles.arcsived}>
                {orderedItems.length && 'Arcsived Shows'}
              </Text>
              <View style={styles.separator} />
            </>
          ) : null}
        </View>
      }
```

Reply:
I think it is a little bit misleading to display the show in the list if it is empty. We can go with your solution, although I would probably show a dislaimer, something like "There are no episodes for this show yet. Check back later maybe?"
What do you think?

Reply:
It was a conscious decision, eg. European Labs is important to have it in the list, but we don't have archives for it. Gyorsnaszád is stream-only but needs the info page. What is more important is the Active / Past shows separation.
As it is, the shows list is definitely very basic and hardly useable on mobile (long scrolling to get to the bottom), besides the search we could have some filters like has-archive maybe? And of course the Active/past anchor link, together with maybe some abc vertical link stripe.
what do you think about these @gammaw @pvj ?